src/utils/image.ts

```diff
@@ -78,32 +78,41 @@ export const getFirebaseStorageResizedImage = (
 // @see https://crypto.stackexchange.com/questions/8533/why-are-bitwise-rotations-used-in-cryptography/8534#8534
 const DIFFUSION_PRIME = 31;
 
-type DetermineAvatar = (options?: {
+type DetermineAvatarProps = {
   avatars?: string[];
   email?: string;
   index?: number;
   partyName?: string;
   pictureUrl?: string;
   userInfo?: firebase.UserInfo;
   user?: User;
-}) => string;
+};
+
+type DetermineAvatarFunc = (
+  options?: DetermineAvatarProps
+) => [string, React.ReactEventHandler<HTMLImageElement>];
```

Comment:
This makes the function backward and forward compatible in the future. Adding and ignoring fields of an object is easier than dealing with array indices.

```suggestion
) => { src: string, onError: React.ReactEventHandler<HTMLImageElement> };
```

Reply:
nice suggestion, like it. 
fixed: https://github.com/sparkletown/sparkle/pull/2760/commits/9cc6e165b25244d4599206263d7e1e47ab5f583f